### DISCUSSION TITLE
Issue with re-attempt of pre/post test in Mobile App

### DISCUSSION THREAD

**Discussion by maddu1** [Sunbird-Ed/Community#458] [Bugs]
Dear Team

Users are facing issues while attempting the pre and post-test on Diksha for Mission Ankur course 4 series.

While attempting the pre-test, after attempting 1st  question the users cannot attempt the next one. We replicated this issue in prod and preprod it was working fine in web portal. the only the issue is in mobile app.

Course Do -id-do_31374196052991180812330

Sharing the screenshots of the same. Please have a look.

![WhatsApp Image 2023-03-22 at 6 21 03 PM](https://user-images.githubusercontent.com/128343712/231729060-782dcc5e-bdf9-4124-9f09-a43b589236b4.jpeg)
![WhatsApp Image 2023-03-22 at 6 21 03 PM (1)](https://user-images.githubusercontent.com/128343712/231729111-19d412c3-302e-4aa1-91d4-8002726dd9ae.jpeg)
![WhatsApp Image 2023-03-22 at 6 21 02 PM](https://user-images.githubusercontent.com/128343712/231729147-188a47ff-f563-40d3-8c09-a1b70bccd62e.jpeg)
![WhatsApp Image 2023-03-22 at 6 21 02 PM (2)](https://user-images.githubusercontent.com/128343712/231729198-b81ad79a-43d1-4337-a643-5713c7d5805f.jpeg)
![WhatsApp Image 2023-03-22 at 6 21 02 PM (1)](https://user-images.githubusercontent.com/128343712/231729252-a0f647c8-e740-4ba9-b1ba-ae23c5c24e70.jpeg)
![WhatsApp Image 2023-03-22 at 6 21 01 PM](https://user-images.githubusercontent.com/128343712/231729283-f72c0aa4-069a-425d-a19b-42d700b4a0d2.jpeg)

**Comment by chitranshu-keshav** [comment]
@maddu1  The information provided in the thread is not sufficient to provide assistance on issue that you have reported. Can you please provide the below details so that the community can help you out with resolving the issue

1) Issue is being faced in which version of Sunbird ED?
2) Is this issue faced only with a specific content or all content of the content type?
3) Is it being faced in all the mobile devices or a specific mobile device ?
4) Is it happening for a specific version of mobile OS ?
5) In which version of ED mobile app is this issue replicated?
6) Share the logs and the analysis done so far by your team and call out what specific help you need.

**Comment by chitranshu-keshav** [comment]
@maddu1 A reminder to get back on the above

**Reply by maddu1** [reply]
Hi @chitranshu-keshav,

Please find the below details. 

1.  Issue is being faced in which version of Sunbird ED?
     5.0.1

2.  Is this issue faced only with a specific content or all content of the content type?
    Specific content in Diksha for Mission Ankur course 4 series [Course Do -id-do_31374196052991180812330]

3.  Is it being faced in all the mobile devices or a specific mobile device ?
     All the mobile devices

4.  Is it happening for a specific version of mobile OS ?
     Both Android & IOS

5.  In which version of ED mobile app is this issue replicated?
     5.1

6.  Share the logs and the analysis done so far by your team and call out what specific help you need.

      At the moment we don't have the access to share the logs. Please find attached the video of the issue. While attempting the pre-test, after attempting 1st  question we cannot attempt to next one. 

[Video.zip](https://github.com/Sunbird-Ed/Community/files/11259046/Video.zip)

**Reply by swayangjit** [reply]
@maddu1  We need the logs of  whether the progress is synced or not for that specific content and also I am unable to understand
Issue is being faced in which version of Sunbird ED?
5.0.1
In which version of ED mobile app is this issue replicated?
5.1
The versions are contradicting here.
